docs/source/index.rst

```diff
@@ -4,7 +4,7 @@
    contain the root `toctree` directive.
 
 
-.. image:: https://raw.githubusercontent.com/kedro-org/kedro/develop/static/img/kedro_banner.png
+.. image:: https://raw.githubusercontent.com/kedro-org/kedro/chore/update-docs-font-family/static/img/kedro_banner.png
```

Comment:
Is this the permanent home? It's probably really picky but it would be nice if the URL didn't include "chore" 😆

Reply:
Awesome, sounds like a plan!

Reply:
oh actually this is from the other branch, I'll remove it